src/server.ts

```diff
@@ -90,40 +88,94 @@ setInterval(() => {
 }, 1000 * 60);
 */
 
+const enum SecondsSinceEpochBrand {}
+type SecondsSinceEpoch = number & SecondsSinceEpochBrand
+
+const enum DateTimeStringBrand { _ = "" }
```

Comment:
This whole tag pattern is inspired by https://basarat.gitbook.io/typescript/main-1/nominaltyping – I'm sure someone at Microsoft could point us in a better direction, but this does help! Check out the commit message for more deets.

Reply:
Thanks -- this is a good enough (and not overly complicated) approach for me :)